README.md

```diff
@@ -55,9 +55,9 @@ For help, or just to chat with fellow mintapi users, please [join us on Discord]
 
 ## Complete Setup
 ### MFA Authentication Methods
-You can handle MFA in one of two ways: email or TOTP
+You can handle MFA in one of four ways: text, voice, email, or TOTP
```

Comment:
I think this statement should indicate that MINT provides these as options for MFA.  It's a little confusing to say "you can handle MFA in on of four ways", but then say, you can't do this or that one, though, here.

Reply:
Changed the wording to draw a finer distinction between what is offered by Mint and what mintapi accepts.